CODEOWNERS

```diff
@@ -11,6 +11,10 @@
 /arch/arm/                                @MaureenHelm @galak
 /arch/arm/core/cortex_m/cmse/             @ioannisg
 /arch/arm/include/cortex_m/cmse.h         @ioannisg
+/arch/common/                             @andrewboie @ioannisg @andyross
+/arch/x86_64/                             @andyross
+/soc/arc/snps_*/                          @vonhust @ruuddw
+/soc/nios2                                @nashif @wentongwu
```

Comment:
`/soc/nios` was missing a trailing slash, found later the next time someone added a file. Fixed in 
commit ca4a078bccc3

The CI check used to had a "performance bug": it would run even when there was no new file. Now it's "optimized too much" so it didn't catch this error :-(

https://github.com/zephyrproject-rtos/ci-tools/commit/fa8374b0a2350639e3#diff-341960e57e998d9240b6ba2d8c7d2980R510

The simplest workaround when editing CODEOWNERS  is probably to add a dummy file and run check_compliance.py locally. Then remove the dummy file.
Or temporarily tweak the code linked above.